### DISCUSSION TITLE
Get data of shop

### DISCUSSION THREAD

**Discussion by Mikey2812** [etsy/open-api#1059] [Q&A]
Hello, i want to know the shop is suppend or not and i want get message of shop. Please help me for the api please. Thanks for all

**Comment by Gareth064** [comment]
The Shop endpoints will tell you the shop message.

As for knowing if a shop is suspended. I believe the shop won't get returned by the API if it's suspended.

Best to ask Etsy developers for a certain answer

developer@etsy.com

**Comment by emergingdzns** [comment]
If the shop is suspended (by Etsy for some violation or something) you won't get it returned by the API. If it's in vacation mode, where the shop owner is suspending operations temporarily, it will still appear in the API. Sadly though, there's no designation in the shop data that will tell you if the shop is in vacation mode. We look at the active_listings count and some other factors to make an assumption.
